src/IO.go

```diff
@@ -67,13 +66,22 @@ func makeDir(name string) {
 	}
 }
 
-func makeRed(input string) string {
+func colorize(input string) string {
 	tokens := strings.Split(input, "\n")
 	for i, row := range tokens {
-		if row == "" {
+		if isBlank(row) {
 			continue
 		}
 		tokens[i] = color.RedString(row)
 	}
 	return strings.Join(tokens, "\n")
 }
+
+func isBlank(input string) bool {
```

Comment:
there is a go function called strings.TrimSpace which does exactly what you are doing here.

strings.TrimSpace(" \t\n a lone gopher \n\t\r\n")